ownCloud/Client/Actions/EditDocumentViewController.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  EditDocumentViewController.swift
+//  ownCloud
+//
+//  Created by Matthias Hühne on 22.01.20.
+//  Copyright © 2020 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2020, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+import ownCloudSDK
+import QuickLook
+
+@available(iOS 13.0, *)
+protocol EditDocumentViewControllerDelegate: class {
+	func editDocumentViewControllerDidDismiss(_ controller: EditDocumentViewController)
+}
+
+@available(iOS 13.0, *)
+class EditDocumentViewController: QLPreviewController, Themeable {
+
+	weak var core: OCCore?
+	var item: OCItem
+	var editDelegte: EditDocumentViewControllerDelegate?
+	var handleSaving: QLPreviewItemEditingMode?
```

Comment:
@hosy `handleSaving` sounds like boolean flag, may be just call it `editingMode` or `savingMode`?